src/index.html

```diff
@@ -6,6 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="dns-prefetch shortcut icon" href="https://monzo.com/static/images/favicon.png">
   <title>Monzo Web</title>
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600" rel="stylesheet">
```

Comment:
Wasn't this served locally before?

Reply:
Trying to get more control over what we need as here it's only the 400 and 600 weight